src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const discountOfSevenDays = 50;
+  const discountOfThreeOrMoreDays = 20;
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - discountOfSevenDays;
+  }
 
+  if (days >= 3 || days >= 6) {
+    return total - discountOfThreeOrMoreDays;
+  }
+
+  return 80;
```

Comment:
The hardcoded return value `80` is incorrect. It should return the `total` cost calculated from the number of days rented without any discounts if the number of days does not qualify for either discount.